src/ipfs-on-path/scripts/ipfs.sh

```diff
@@ -1,5 +1,7 @@
 #!/usr/bin/env bash
 
+export IPFS_PATH="/Users/henriquedias/Desktop/.ipfs"
```

Comment:
We can ask all users to rename to `henriquedias` :upside_down_face:  but maybe read the value from ipfs-desktop's `config.json`/`ipfsConfig.path` instead ;)

Reply:
😆 This was... unfortunate and funny. The thing is: from where should we read this? How to do it on windows? How to parse a json file? Where is the json file? There's lot of variables we don't know and we'd always need an hardcoded thing. Perhaps we could have a file within that directory that is gitignored and contains the path. What do you think?